File: src/CustomSDK/Examples/CPlusPlus/CustomModules/CustomModule/src/CustomModule.cpp
#include "stdafx.h"

#include "CustomModule/CustomModule.h"

#include <iostream>

CustomModule::CustomModule():
  AbstractCustomModule()
{
}

CustomModule::~CustomModule()
{
}

void CustomModule::doSayHello()
{
  std::cout <<  "[CPlusPlus]:CustomModule::Hello World!" << std::endl;
}

File: src/CustomSDK/Examples/Java/CustomModules/CustomModule4Java/src/CustomModule4Java/CustomModule.java
package CustomModule4Java;

import CustomFramework4Java.AbstractCustomModule;
import CustomFramework4Java.LogManager;

public class CustomModule extends AbstractCustomModule
{
  public CustomModule()
  {
    super();
  }

  @Override
  public void doSayHello()
  {
    System.out.println("[Java]:CustomModule::Hello World!");
  }

  public static void doRegister()
  {
    LogManager.getInstance().trace("[Java]:"+ CustomModule.class.getName() +" Registered...");
  }
}
